components/IssueCardColumn.vue

```diff
@@ -0,0 +1,24 @@
+<script lang="ts" setup>
+import { VueDraggable } from 'vue-draggable-plus'
+
+defineProps<{ title: string; modelValue: unknown[] }>()
+</script>
+
+<template>
+  <div
+    class="flex min-w-[350px] max-w-[350px] flex-1 flex-col rounded-[4px] border border-rad-border-hint bg-rad-background-dip"
+  >
+    <h3 class="p-2 text-xl font-semibold capitalize">{{ title }}</h3>
+
+    <VueDraggable
+      tag="ul"
+      class="flex flex-1 flex-col gap-2 p-2"
+      v-bind="$attrs"
+      :model-value="modelValue"
```

Comment:
I'm not entirely sure what it is I'm doing in this line.

I tried using the fallthrough attributes pattern as described in [the docs](https://vuejs.org/guide/components/attrs.html#attribute-inheritance-on-multiple-root-nodes), which worked in practice as the drag and drop was working, but typescript was complaining about `model-value` missing.

I landed on [this stackoverflow thread](https://stackoverflow.com/questions/76369215/v-model-fallthrough-attribute-on-component-child-not-working), and after some experimenting I ended up with this which keeps typescript happy.

@maninak Do you have any experience with this? Do you know of a better way to handle it?

Reply:
Hey, your gut was right, what was happening there was very hacky.

The issue was that instead of binding the necessary props (i.e. `v-model`, which is actually a directive, not a prop as well as the prop `group`) to `<VueDraggable>` directly, we were binding them on its parent (i.e.  `<IssueCardColun>`) and then practically "piping through" those props finally on to `<VueDraggable>` with `v-bind=$attrs`.